resources/function_help/json/overlay_intersects

```diff
@@ -72,6 +82,14 @@
     {
       "expression": "overlay_intersects(layer:='regions', min_inscribed_circle_radius:=0.54)",
       "returns": "true if the current feature spatially intersects a region and the intersection area maximum inscribed circle's radius (of at least one of the parts in case of multipart) is greater or equal to the 0.54"
+    },
+    {
+      "expression": "overlay_intersects(layer:='regions', expression:= geom_to_wkt($geometry), return_details:=true)",
+      "returns": "an array of maps with the feature id, the expression value, the overlap value and the radius of the maximum inscribed circle"
+    },
+    {
+      "expression": "overlay_intersects(layer:='regions', expression:= geom_to_wkt($geometry), sort_by_intersection_size:=true)",
```

Comment:
```suggestion
      "expression": "overlay_intersects(layer:='regions', expression:= geom_to_wkt($geometry), sort_by_intersection_size:='des')",
```
Should it be 'des' or 'desc'?

Reply:
the code really checks for startWith 'asc' des' case insensitive so both 'des' and 'desc' will work, if you think it's clearer to use 'desc' in the function doc I have no objections.

Reply:
I was suggesting that because asc/desc is, I think, commonly known by SQL users...
> the code really checks for startWith 'asc' des'

Re "startswith", is there any way users are aware of this feature (other than reading the code or this thread)? It's nice to give them freedom (i see the advantages for localization, here) but if nobody knows, I'm afraid they will likely only use the exact matching. May be an example with other spellings, clarify the description or just stick to exact matching? Sorry to/if I annoy you with this.
And for my knowledge, anyone knows if we already have startswith parameters? I can see how/if we made it obvious to users and if necessary, try to improve their help or open a report...